tox.ini

```
@@ -53,9 +53,20 @@ commands =
     --out-dir {toxinidir}/dist/ \
     {toxinidir}
 
+# deprecated: use more generic 'lint' instead
 [testenv:flake8]
-deps = flake8
-commands = flake8
+deps = {[testenv:lint]deps}
+envdir = {toxworkdir}/lint
+skip_install = true
+commands =
+  python -m pre_commit run -a flake8
```

Plz use the long version of the arg:
```suggestion
  python -m pre_commit run --all-files flake8
```